src/components/categories/money/Money.vue

```diff
@@ -0,0 +1,262 @@
+                        text: this.$t('money.bot.chooseTime'),
+                        style: "default"
+                    });
+                    this.showTimeButtons = true;
+                }
+            },
+            showExchangeAndButtons(exchange) {
+                this.showExchanges = false;
+                this.exchange = exchange.symbol;
+                this.$emit('addMessage', {
+                    author: "user",
+                    text: `${this.$t('money.user.chosenExchange')} ${exchange.name}`,
+                    style: "default"
+                });
+                this.$emit('addMessage', {
+                    author: "bot",
+                    text: this.$t('money.bot.chooseTime'),
+                    style: "default"
+                });
+                this.showTimeButtons = true;
+            },
+            showHistoryData() {
+                this.showTimeButtons = false;
+                this.$emit('addMessage', {
+                    author: "user",
+                    text: `${this.$t('money.user.chosenHistory')}`,
+                    style: "default"
+                });
+                this.$emit('addMessage', {
+                    author: "bot",
+                    text: this.$t('money.bot.historyChoice'),
+                    style: "default"
+                });
+                this.choosePeriod = true;
+            },
+            showDayChoiceComponent() {
+                this.choosePeriod = false;
+                this.$emit('addMessage', {
+                    author: "user",
+                    text: `${this.$t('money.user.chosenDay')}`,
+                    style: "default"
+                });
+                this.$emit('addMessage', {
+                    author: "bot",
+                    text: this.$t('money.bot.dayChoice'),
+                    style: "default"
+                });
+                this.showDayChooser = true;
+            },
+            showPeriodChoiceComponent() {
+                this.choosePeriod = false;
+                this.$emit('addMessage', {
+                    author: "user",
+                    text: `${this.$t('money.user.chosenPeriod')}`,
+                    style: "default"
+                });
+                this.$emit('addMessage', {
+                    author: "bot",
+                    text: this.$t('money.bot.periodChoice'),
+                    style: "default"
+                });
+                this.showPeriodChooser = true;
+            },
+            showActualData() {
+                this.showTimeButtons = false;
+                this.$emit('addMessage', {
+                    author: "user",
+                    text: `${this.$t('money.user.chosenActual')}`,
+                    style: "default"
+                });
+                this.$emit('addMessage', {
+                    author: "bot",
+                    text: this.$t('money.bot.actualData'),
+                    style: "default"
+                });
```

Comment:
Do formattera przenieśmy formatowanie daty i reużywajmy go wszędzie, gdzie jest to potrzebne, aby zachować spójność formatu daty.